CMakeLists.txt

```diff
@@ -1,5 +1,12 @@
-cmake_minimum_required(VERSION 3.13)
-cmake_policy(SET CMP0144 NEW)
+cmake_minimum_required(VERSION 3.15)
+if (POLICY CMP0167)
+    # deprecated by definition, TODO: port project to "Modern CMake" https://gist.github.com/mbinna/c61dbb39bca0e4fb7d1f73b0d66a4fd1
+    cmake_policy(SET CMP0167 OLD) 
+endif()
+if (POLICY CMP0144)
+    cmake_policy(SET CMP0144 NEW)
+endif()
```

Comment:
Am I right in understanding that this is just a fix, since we have this policy only since version 3.27?

At the same time, if someone tries to use an older version now, they will get a warning or even an error?

Reply:
> Am I right in understanding that this is just a fix, since we have this policy only since version 3.27?

Yes.

> At the same time, if someone tries to use an older version now, they will get a warning or even an error?

I used to think that the behavior of policies was like: if I know about this policy I will set the right one, if I don't know I don't care.  But when I set the CMP0167 policy as `OLD` I got an error in CI. I think with `NEW` there was a warning.